### PR TITLE
Add rspec-rails 4.0.0 documentation

### PR DESCRIPTION
**Pull request by benoittgt**
- https://github.com/rspec/rspec-rails/pull/2298

Capture d’écran 2020-03-24 à 23 11 16

---

Capture d’écran 2020-03-24 à 23 19 43

### CODE REVIEW

**Comment by benoittgt** [comment]
I think it can be merged.

File: source/documentation/4.0/rspec-rails/file.Changelog.html [hunk 1]
```diff
@@ -58,7 +58,18 @@
...
+<p><a href="https://github.com/rspec/rspec-rails/compare/v4.0.0...4-0-development">Full Changelog</a></p>
+
+<p>Bug Fixes:</p>
+
+<ul>
+<li>Remove warning when calling <code>driven_by</code> in system specs. (Aubin Lorieux, #2302)</li>
+<li>Fix comparison of times for <code>#at</code> in job matchers. (Jon Rowe, Markus Doits, #2304)</li>
+</ul>
+
+<h3>4.0.0 / 2020-03-24</h3>
```
Comment (JonRowe) [inline source/documentation/4.0/rspec-rails/file.Changelog.html]: This shouldn't be here until released, might need to run after a `git checkout v4.0.0`

Comment (benoittgt) [reply source/documentation/4.0/rspec-rails/file.Changelog.html]: This is what I do but the release was on [`d810df5`](https://github.com/rspec/rspec-rails/releases/tag/v4.0.0) and the rake command _pull_ to the last commit from `v4.0.0`. It is the normal behavior of the command. For this time I will skip the `git pull` and stick to the commit from the release.

File: source/documentation/4.0/rspec-rails/file.Changelog.html [hunk 2]
```diff
@@ -58,25 +58,65 @@
         <div class="clear"></div>
       </div>
 
-      <div id="content"><div id='filecontents'><h3>Development</h3>
+      <div id="content"><div id='filecontents'><h3>4.0.0.rc1 / 2020-03-13</h3>
```
Comment (benoittgt) [inline source/documentation/4.0/rspec-rails/file.Changelog.html]: This is not what we want 🤔

Comment (JonRowe) [reply source/documentation/4.0/rspec-rails/file.Changelog.html]: Nope :joy:, you might need to manually grab the commit after. The changelog in the release wasn't cleaned, my mistake :/

Comment (benoittgt) [reply source/documentation/4.0/rspec-rails/file.Changelog.html]: Ahah. Yes it was the next one. It should be good now. 😊

File: source/documentation/4.0/rspec-rails/file.Changelog.html [hunk 3]
```diff
@@ -58,18 +58,7 @@
...
-
-<p>Bug Fixes:</p>
-
-<ul>
-<li>Remove warning when calling <code>driven_by</code> in system specs. (Aubin Lorieux, #2302)</li>
-<li>Fix comparison of times for <code>#at</code> in job matchers. (Jon Rowe, Markus Doits, #2304)</li>
-</ul>
-
-<h3>4.0.0 / 2020-03-24</h3>
+      <div id="content"><div id='filecontents'><h3>4.0.0 / 2020-03-24</h3>
```
Comment (benoittgt) [inline source/documentation/4.0/rspec-rails/file.Changelog.html]: YES 🎇

**Comment by benoittgt** [comment]
Good question. I opened an issue on rdoc https://github.com/ruby/rdoc/issues/759

**Comment by JonRowe** [comment]
Do you want to add some white space trimming to the bash command?

**Comment by benoittgt** [comment]
That's not a bad idea. If I can fix it on `rdoc` it could be great too but maybe a new `rdoc` release will not be soon.

If kids are sleeping enough tomorrow I handle it :)

**Comment by benoittgt** [comment]
It takes some times to fix on rdoc/erb. In the meantime I'm gonna add a bash command. :)

**Comment by benoittgt** [comment]
For the white space. https://github.com/rspec/rspec.github.io/pull/139

**Comment by benoittgt** [comment]
I had to skip the git checkout command from the rake task for https://github.com/rspec/rspec.github.io/pull/136#discussion_r400452243

Changelog seems ok

Capture d’écran 2020-04-08 à 23 25 00

No blank lines.

**Comment by JonRowe** [comment]
I'm happy if you're happy.

**Comment by pirj** [comment]
Not deployed yet http://rspec.info/documentation/4.0/rspec-rails/ 😕

**Comment by benoittgt** [comment]
Yep :)

**Pull request merged**